core/lib/Thelia/Core/Template/Loop/BaseSpecificModule.php

```diff
@@ -28,6 +28,7 @@
  * {@inheritdoc}
  * @method int getId()
  * @method int[] getExclude()
+ * @method int[] getExcludeCode()
```

Comment:
Hello,
The typage is `string[]` not `int[]`

Reply:
Nice catch :) Thanks.